packages/navie/src/services/anthropic-completion-service.ts

```diff
@@ -132,39 +155,59 @@ export default class AnthropicCompletionService implements CompletionService {
   }
 
   async *complete(messages: readonly Message[], options?: { temperature?: number }): Completion {
-    try {
-      const model = this.buildModel(options);
-
-      const sentMessages = mergeSystemMessages(messages);
-      for (const message of sentMessages) this.trajectory.logSentMessage(message);
-
-      const response = await model.stream(sentMessages.map(convertToMessage));
+    const usage = new Usage(COST_PER_M_TOKEN[this.modelName]);
+    const model = this.buildModel(options);
+    const sentMessages = mergeSystemMessages(messages);
+    for (const message of sentMessages) this.trajectory.logSentMessage(message);
 
-      const usage = new Usage(COST_PER_M_TOKEN[this.modelName]);
+    const maxAttempts = 5;
```

Comment:
Can this be an env var? Maybe used in other retry situations too.

Reply:
With GPT 1o we will also want substantial retry back off.

Reply:
We should update https://github.com/getappmap/appmap-js/pull/2000 as well (cc @dividedmind)